content/en/platform/corda/5.2/developing-applications/cordapp-template/reset.md

```diff
@@ -35,81 +35,22 @@ The instructions in this section use the following terms:
 
-1. Stop any running combined worker processes:
-   * On Linux/macOS:
+1. Run the following command, to stop and remove Docker containers and network used by the Corda cluster, where:
+   * `config/combined-worker-compose.yaml` is the path to the compose file. This should match the value of the `composeFilePath` property in the `cordaRuntimeGradlePlugin` section of the `build.gradle` file.
+   * `corda-cluster` should match the `composeNetworkName` property. `corda-cluster` is the default value and is not set in the template code.
 
-      a. To find the process ID (pid), run:
-
-      ```shell
-      ps -ef | grep corda-combined-worker
-      ```
-
-       The combined worker process is the second ID returned. For example, 63892 in the following response:
-
-      ```shell
-      503 63892 52310   0  1:05PM ??         2:36.96 /Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home/bin/java -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005 -Dco.paralleluniverse.fibers.verifyInstrumentation=true -jar /Users/.gradle/caches/modules-2/files-2.1/net.corda/corda-combined-worker/{{<version-num>}}.0.0.0/1b7c6fdecd5e54e09ec080905f2b6e14fad1a4d5/corda-combined-worker-{{<version-num>}}.0.0.jar --instance-id=0 -mbus.busType=DATABASE -spassphrase=password -ssalt=salt -spassphrase=password -ssalt=salt -ddatabase.user=user -ddatabase.pass=password -ddatabase.jdbc.url=jdbc:postgresql://localhost:5432/cordacluster -ddatabase.jdbc.directory=/Users/.corda/corda5/jdbcDrivers
-      ```
-
-      Alternatively, you can use the jps command to list java processes running:
-
-      ```shell
-      jps
-     ```
-     Which returns:
-     ```
-     23666 GradleDaemon
-     25400 GradleDaemon
-     18986
-     1114
-     41770 Jps
-     25419 KotlinCompileDaemon
-     63892 corda-combined-worker-{{<version-num>}}.0.0.jar
-
-     ```
-
-      b. To stop the process, run:
-
-      ```shell
-      kill <pid-for-corda-combined-worker>
-      ```
-
-   * On Windows, run in PowerShell:
-     ```shell
-     Invoke-CimMethod -Query "SELECT * from Win32_Process WHERE name LIKE 'java.exe' and Commandline like '%corda-combined-worker%'" -MethodName "Terminate"
-     ```
-2. Check if the above commands were successful:
-   * On Linux/macOS, run:
-      ```shell
-      ps -ef | grep corda-combined-worker
-      ```
-      or
-      ```shell
-      jps
-      ```
-
-   * On Windows, run:
-      ```shell
-      Get-CimInstance -Query "SELECT * from Win32_Process WHERE name LIKE 'java.exe' and Commandline like '%corda-combined-worker%'"
-      ```
-3. To stop any `CSDEpostgresql` containers, run:
-   ```shell
-   docker stop CSDEpostgresql
-   ```
-   You can check which containers are running using:
    ```shell
-   docker ps
+   docker compose -f config/combined-worker-compose.yaml -p corda-cluster down
    ```
-4. Remove the container:
-   ```shell
-   docker rm CSDEpostgresql
-   ```
-5. Run the Gradle clean task:
+   
+1. Run the Gradle clean task:
```

Comment:
Numbering in the list has reset...

Reply:
The numbering should be rendered ok int he preview?